### PR TITLE
GUT-424 Added new variation "extended" for Pullquote block

### PR DESCRIPTION
**Pull request by sannysrivastava**
### Make sure you complete these items:

- [X] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [X] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))
- [ ] If adding a new pattern, in the PR comment, included a screenshot and link to the static Vercel deployment
- [ ] If changes to build scripts or the Node.js server, tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw)
- - If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

File: packages/larva-patterns/modules/pullquote/pullquote.extended.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const clonedeep = require('lodash.clonedeep');
+
+const pullquote_prototype = require( './pullquote.prototype' );
+const pullquote_extended = clonedeep( pullquote_prototype );
+
+pullquote_extended.pullquote_classes += ' extended';
+
+module.exports = pullquote_extended;
```
Comment (rsalvucci) [inline packages/larva-patterns/modules/pullquote/pullquote.extended.js:8]: There is an error in this file, it needs a blank line.

**Comment by rsalvucci** [comment]
The package-lock.json files should be removed, unless you were updating some dependencies. Also why is there a lot of Billboard Español token files, hen your change is only to the pull quote block?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unpublished Changes
+## 10:00:0 08-05-2022
+* larva-patterns - Add `extended` variant for `pullquote` module.
```
Comment (vtellapmc) [inline CHANGELOG.md:9]: This need to be under `Unpublished Changes`

File: packages/larva-patterns/modules/pullquote/pullquote.extended.js (hunk 1 above)
Comment (sannysrivastava) [reply packages/larva-patterns/modules/pullquote/pullquote.extended.js]: Resolved, not sure why it did not flagged in `npm run lint`

File: CHANGELOG.md (hunk 2 above)
Comment (sannysrivastava) [reply CHANGELOG.md:8]: Fixed it.

**Comment by sannysrivastava** [comment]
> 



> The package-lock.json files should be removed, unless you were updating some dependencies. Also why is there a lot of Billboard Español token files, hen your change is only to the pull quote block?

I tried to replace the package-lock.json from master but still shows under changed files, my changes are only to the pullquote.extented.js file but `npm run prod` created other changes

**Comment by rsalvucci** [comment]
Can you revert the bbesp token files to the master branch? Even if the package-lock.json files show under changed files those files should still be reverted. It is most likely that they changes are just in the hash numbers, but if you are running a different version of anything locally that would get changed in Larva and could introduce issues.

**Comment by sannysrivastava** [comment]
> Can you revert the bbesp token files to the master branch? Even if the package-lock.json files show under changed files those files should still be reverted. It is most likely that they changes are just in the hash numbers, but if you are running a different version of anything locally that would get changed in Larva and could introduce issues.

I have reverted the files, please check again.

**Review by rsalvucci — APPROVED** [review]

**Pull request merged**